mitxgraders/baseclasses.py

```diff
@@ -97,7 +96,7 @@ def coerce2unicode(obj):
         with other references remaining intact.
         """
         if isinstance(obj, six.string_types):
```

Comment:
Sorry to disappoint you, but if you're going to remove python 2 support, then the thing to do is to remove the `six` library entirely. `six` exists to provide a compatibility layer between python 2 and 3 (because 2*3=6). You should start by removing six from requirements.txt (in both versions), and then handling the fallout.

Reply:
I will do that in another PR to make it easier to review.